tools/exporter/exporter.py

```diff
@@ -497,6 +518,55 @@ def parse_command_line_arguments(logger):
 
     return config_filename, export_formats, args.list_export_profiles, loop_enabled
 
+def initial_setup(logger):
+    """
+    Creates a new directory in current working directory called 'iAuditor Audit Exports'. Default config file placed
```

Comment:
What if you run setup twice and the directory exists?

Reply:
If directory exists, line 537 'os.makedirs()` fails within the try statement and exits the setup. The exception is written to console informing user that directory already exisits.